docs/notes/bugfix-14402.md

```diff
@@ -0,0 +1 @@
+#    mobileRestorePurchases enhancement request - no restores available
```

Comment:
This needs more of a description of what this feature is doing - since it is an enhancement ensure it is explained appropriately in the release note.